### PR TITLE
add html-css Google popup

### PR DESCRIPTION
**Pull request by AlexNugda**
### Html-Css popup

[Demo](https://alexnugda.github.io/html-css-popup/) |
[Code base](https://github.com/AlexNugda/frontend-2019-homeworks/tree/html-css-popup/submissions/AlexNugda/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/AlexNugda/html-css-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,252 @@
+html, body{
+	margin: 0px;
```
Comment (A-Ostrovnyy) [inline submissions/AlexNugda/html-css-popup/style.css]: Use Prettier for formatting your code.

File: submissions/AlexNugda/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+	display: flex;
+	justify-content: flex-end;
+	padding: 20px;
+}
+
+.google-menu-container a{
+    color: #212121;
+}
+
+ul.google-menu-list-main, ul.google-menu-list{
```
Comment (A-Ostrovnyy) [inline submissions/AlexNugda/html-css-popup/style.css]: Avoid styling by tag. In this case simple class `.google-menu-list-main` will be enough.

File: submissions/AlexNugda/html-css-popup/style.css (hunk 1 above)
Comment (AlexNugda) [reply submissions/AlexNugda/html-css-popup/style.css]: Done

File: submissions/AlexNugda/html-css-popup/style.css (hunk 2 above)
Comment (AlexNugda) [reply submissions/AlexNugda/html-css-popup/style.css]: Done

**Comment by A-Ostrovnyy** [comment]
Nice work!

**Pull request merged**